tests/flow/test_function_calls.py

```diff
@@ -2194,4 +2194,50 @@ def test85_division_inputs(self):
         try:
             actual_result = graph.query(query)
         except redis.ResponseError as e:
-            self.env.assertContains("Division by zero", str(e))
\ No newline at end of file
+            self.env.assertContains("Division by zero", str(e))
+
+    # GitHub Issue #2291.
+    # Ensure that a node can be added to a list.
+    def test59_node_add_to_array(self):
```

Comment:
This is not the best place for these particular tests. I'll find a better place for those.